ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -343,3 +382,267 @@ Set RHODS Users Group To Inexistent Group
+    Should Be Equal As Integers    ${rc}     0
+
+Create An Accelerator Profile Via UI
+    [Documentation]    Fill Accelerator Profile values with the ones provided in the arguments and click on
+    ...                create button
+    ...                view: Create Accelerator profile
+    [Arguments]   ${name}  ${identifier}  ${description}=${EMPTY}  ${enabled}=True
+    ...           ${tolerations}=${EMPTY}  ${tol_operator}=Equal  ${tol_effect}=None  ${tol_key}=key
+    ...           ${tol_value}=value  ${tol_seconds}=Forever
+    Wait Until Page Contains Element  ${ACCELERATOR_NAME}  timeout=5s
+    Input Text    ${ACCELERATOR_NAME}    ${name}
+    Input Text    ${ACCELERATOR_IDENTIFIER}    ${identifier}
+    IF    "${description}" != "${EMPTY}"
+        Input Text    ${ACCELERATOR_DESCRIPTION}    ${description}
+    END
+    # By default are enabled
+    IF  ${enabled} != True
+        Click Element    ${ACCELERATOR_ENABLED_SWITCH}
+    END
+    # Tolerations
+    IF    "${tolerations}" != "${EMPTY}"
+        Click Button    ${ACCELERATOR_ADD_TOLERATION_BTN}
+        Wait Until Element Is Visible    ${ACCELERATOR_ADD_TOLERATION_MODAL_FORM}
+        # Select Operator
+        Click Button    ${ACCELERATOR_TOLERATION_OPERATOR_DROPDOWN}
+        Click Element    //div[@data-testid="toleration-operator-select"]//div[text()="${tol_operator}"]
+        # Select Effect
+        Click Button    ${ACCELERATOR_TOLERATION_EFFECT_DROPDOWN}
+        Click Element    //label[@for="effect-select"]/parent::*/following::div//*[text()="${tol_effect}"]
+        # Input Key and value
+        Input Text    ${ACCELERATOR_TOLERATION_KEY}    ${tol_key}
+        Input Text    ${ACCELERATOR_TOLERATION_VALUE}    ${tol_value}
+        # Toleration Seconds
+        IF    "${tol_seconds}" != "Forever"
+            Click Element    ${ACCELERATOR_TOLERATION_SECONDS_CUSTOM_RADIO}
+            Clear element and input text    ${ACCELERATOR_TOLERATION_SECONDS_CUSTOM_VALUE}   ${tol_seconds}
+        ELSE
+            Click Element    ${ACCELERATOR_TOLERATION_SECONDS_FOREVER_RADIO}
+        END
+        Click Button    ${ACCELERATOR_TOLERATION_ADD_OR_UPDATE_BUTTON}
+    END
+    Click Button    ${CREATE_OR_UPDATE_ACCELERATOR_PROFILE_DETAIL_BTN}
+
+Modify The Accelerator Profile
+    [Documentation]    Modify Accelerator Profile values with the ones provided in the arguments and click on
+    ...                "Update acccelerator profile" button
+    ...                view: Edit Accelerator profile
+    [Arguments]   ${original_display_name}  ${display_name}=${EMPTY}  ${identifier}=${EMPTY}  ${description}=${EMPTY}
+    ...           ${enabled}=${EMPTY}  ${tolerations}=${EMPTY}  ${tol_operator}=${EMPTY}  ${tol_effect}=${EMPTY}
+    ...           ${tol_key}=${EMPTY}  ${tol_value}=${EMPTY}  ${tol_seconds}=${EMPTY}
+    Wait Until Element Is Visible   //h1[text()="Edit ${original_display_name}"]  timeout=10
```

Comment:
based on the name of the keyword, I'd assume it clicks the button for editing as first step